plugin/src/main/java/io/jenkins/plugins/casc/Attribute.java

```diff
@@ -225,18 +230,73 @@ public CNode describe(Owner instance, ConfigurationContext context) throws Confi
                     ": No configurator found for type " + type);
         }
         try {
-            Object o = getValue(instance);
-            if (o == null) {
-                return null;
+            Object o;
+            if(isJsonSchema) {
+                 o = getType();
+                if (o == null) {
+                    return null;
+                }
+            } else {
+                 o = getValue(instance);
+                if (o == null) {
+                    return null;
+                }
+            }
+
+            // In Export we sensitive only those values which do not get rendered as secrets
+            boolean shouldBeMasked = isSecret(instance);
+            if (multiple) {
+                Sequence seq = new Sequence();
+                if (o.getClass().isArray()) o = Arrays.asList((Object[]) o);
+                if(o instanceof Iterable) {
+                    System.out.println("This is iterable");
+                    for (Object value : (Iterable) o) {
+                        seq.add(_describe(c, context, value, shouldBeMasked));
+                    }
+                }
+                return seq;
+            }
+            return _describe(c, context, o, shouldBeMasked);
+        } catch (Exception | /* Jenkins.getDescriptorOrDie */AssertionError e) {
+            // Don't fail the whole export, prefer logging this error
+            LOGGER.log(Level.WARNING, "Failed to export", e);
+            return new Scalar("FAILED TO EXPORT\n" + instance.getClass().getName() + "#" + name + ": "
+                + printThrowable(e));
+        }
+    }
+
+
+    /**
+     * This function is for the JSONSchemaGeneration
+     * @param instance
+     * @param context
+     * @return
+     */
```

Comment:
Will need to be fixed before the merge

Reply:
The empty Javadoc is still here @sladyn98

Reply:
Yeah this got lost in all the review comments :(